07_memory/user_mem/logs.rst

```diff
@@ -0,0 +1,531 @@
+    Iter No31
+    Start adress is 0x7f71ea32b010
+    Alloc time = 6229ns     Free time = 14941ns
+    Iter No32
+    Start adress is 0x7f716a32b010
+    Alloc time = 5457ns     Free time = 14139ns
+    Iter No33
+    Start adress is 0x7f706a32b010
+    Alloc time = 4822ns     Free time = 14107ns
+    Iter No34
+    Start adress is 0x7f6e6a32b010
+    Alloc time = 4940ns     Free time = 14118ns
+    Iter No35
+    Can't allocate:(
+    Aver allocating time is 240451ns        Aver freeing time is 2582ns
+    ************************************************
+    *****************ALLOCA TESTING*****************
+    ****************size = (2^x)+0******************
+    ************************************************
+    Iter No0
+    Start adress is 0x7fffb2149340
+    Alloc time = 139ns      Free time = 0ns
+    Iter No1
+    Start adress is 0x7fffb2149330
+    Alloc time = 74ns       Free time = 0ns
+    Iter No2
+    Start adress is 0x7fffb2149330
+    Alloc time = 71ns       Free time = 0ns
+    Iter No3
+    Start adress is 0x7fffb2149330
+    Alloc time = 71ns       Free time = 0ns
+    Iter No4
+    Start adress is 0x7fffb2149330
+    Alloc time = 53ns       Free time = 0ns
+    Iter No5
+    Start adress is 0x7fffb2149320
+    Alloc time = 53ns       Free time = 0ns
+    Iter No6
+    Start adress is 0x7fffb2149300
+    Alloc time = 53ns       Free time = 0ns
+    Iter No7
+    Start adress is 0x7fffb21492c0
+    Alloc time = 57ns       Free time = 0ns
+    Iter No8
+    Start adress is 0x7fffb2149240
+    Alloc time = 54ns       Free time = 0ns
+    Iter No9
+    Start adress is 0x7fffb2149140
+    Alloc time = 54ns       Free time = 0ns
+    Iter No10
+    Start adress is 0x7fffb2148f40
+    Alloc time = 53ns       Free time = 0ns
+    Iter No11
+    Start adress is 0x7fffb2148b40
+    Alloc time = 54ns       Free time = 0ns
+    Iter No12
+    Start adress is 0x7fffb2148340
+    Alloc time = 60ns       Free time = 0ns
+    Iter No13
+    Start adress is 0x7fffb2147340
+    Alloc time = 5153ns     Free time = 0ns
+    Iter No14
+    Start adress is 0x7fffb2145340
+    Alloc time = 2554ns     Free time = 0ns
+    Iter No15
+    Start adress is 0x7fffb2141340
+    Alloc time = 4066ns     Free time = 0ns
+    Iter No16
+    Start adress is 0x7fffb2139340
+    Alloc time = 3576ns     Free time = 0ns
+    Iter No17
+    Start adress is 0x7fffb2129340
+    Alloc time = 5675ns     Free time = 0ns
+    Iter No18
+    Start adress is 0x7fffb2109340
+    Alloc time = 3462ns     Free time = 0ns
+    Iter No19
+    Start adress is 0x7fffb20c9340
+    Alloc time = 3277ns     Free time = 0ns
+    Iter No20
+    Start adress is 0x7fffb2049340
+    Alloc time = 3231ns     Free time = 0ns
+    Iter No21
+    Start adress is 0x7fffb1f49340
+    Alloc time = 4879ns     Free time = 0ns
+    Iter No22
+    Start adress is 0x7fffb1d49340
+    Alloc time = 3527ns     Free time = 0ns
+    Iter No23
+    Segmentation fault (core dumped)
```

Comment:
Is it possible to change program for exclude this segmentation fault?

Reply:
Yes, at least we can not refer to the returned pointer.

Reply:
This is not enough.

Reply:
If it is difficult to fix segfault and provide correct result please leave version with segmentation fault.

Reply:
I've done it)